### DISCUSSION TITLE
Delete old builds of branch after merged

### DISCUSSION THREAD

**Discussion by 23kbps** [sbidoul/runboat#86] [General]
Is it possible to make runboat deletes all the old builds of a branch after it's merged?

**Comment by sbidoul** [comment]
There is code in the webhook that does that. Is it not working?

**Reply by 23kbps** [reply]
yeah, after we merged PR, builds of old branch still existing, and also it triggered another build of the base branch

**Reply by sbidoul** [reply]
> and also it triggered another build of the base branch

That is normal.

> after we merged PR, builds of old branch still existing

any chance you can add some tracing around [here](https://github.com/sbidoul/runboat/blob/02a29d895b7fe3d00161e62afef60e058f801575/src/runboat/webhooks.py#L64) in your deployment to understand if this webhook works or not?
